standard/unsafe-code.md

```diff
@@ -967,6 +967,8 @@ In a member access of the form `E.I`, if `E` is of a struct type and a member lo
 
 The subsequent elements of the fixed-size buffer can be accessed using pointer operations from the first element. Unlike access to arrays, access to the elements of a fixed-size buffer is an unsafe operation and is not range checked.
 
+In an element access of the form `E[I]`, if `E` is a fixed-sized buffer, and `I` is implicitly convertible to `int`, then `E[I]` is evaluated exactly as the *pointer_element_access* (§22.6.4), `P[I]`, where `P` is the address of the fixed-sized buffer, `E`.
```

Comment:
I *think* this is in the wrong place, the change is restricted to expressions of the form `E.I[J]` – compare the two examples in the proposal document.

I suggest where this special case/feature needs to be added is in the third bullet above (line 965) which becomes something following the *outline*:

- Otherwise, if `E` is a moveable variable ([§22.4](unsafe-code.md#224-fixed-and-moveable-variables)) then:
  - if the expression `E.I` is a *fixed_pointer_initializer* ([§22.7](unsafe-code.md#227-the-fixed-statement)) everything is good
  - otherwise if the expression `E.I` is a *primary_no_array_creation_expression* within an *element_access* expression of the form `E.I[J] then yada yada;
  - otherwise a compile-time error occurs.

I said it was an *outline*!

Does this capture this feature?

Reply:
Probably should have said that in `E.I[J]` the `E.` can of course be the implicit `this.`